deploy/values.yaml

```diff
@@ -148,24 +148,40 @@ application:
 #####################
 mongodb:
 
-  fullnameOverride: review-mongodb  
+  fullnameOverride: review-mongodb
 
   updateStrategy:
     type: Recreate
 
-  resources: 
+  resources:
      limits:
-       memory: 1Gi 
+       memory: 1Gi
        cpu: 0.5
      requests:
        memory: 128Mi
        cpu: 0.1
-       
+
   auth:
     enabled: true
     # Its created via an external secret from vault; defined above
     existingSecret: review-mongodb-creds
 
+  readinessProbe:
```

Comment:
we can update probes for values-local and not here since probes are an issue on snos only.